### PR TITLE
Adding hvpy Python package

### PR DESCRIPTION
**Pull request by nabobalis**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/hvpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  home: https://hvpy.readthedocs.io/en/latest/
+  summary: Helioviewer Python API Wrapper
+  license: BSD-2-Clause
+  license_file: LICENSE
+  doc_url: https://hvpy.readthedocs.io/en/latest/
+  dev_url: https://github.com/Helioviewer-Project/python-api
+
+extra:
+  recipe-maintainers:
+    - dgarciabriseno
```
Comment (nabobalis) [inline recipes/hvpy/meta.yaml:45]: @dgarciabriseno do you approve?

File: recipes/hvpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - wafels
```
Comment (nabobalis) [inline recipes/hvpy/meta.yaml:46]: @wafels do you approve?

Comment (wafels) [reply recipes/hvpy/meta.yaml:45]: I approve

**Review by wafels — COMMENTED** [review]
I approve.

**Comment by dgarciabriseno** [comment]
I also approve.

**Comment by nabobalis** [comment]
Hello, can someone from @conda-forge/help-python, when you have time review this recipe please!

**Review by dopplershift — APPROVED** [review]

**Comment by nabobalis** [comment]
Thank you @dopplershift

**Pull request merged**